Corrections to video encoding guide & add FFmpeg instructions. (#6065)
ppy/osu-wiki: wiki/Guides/Compressing_files/en.md

![Removing audio tracks from the audio tab in Handbrake](img/removeaudio-handbrake.jpg "Removing the audio tracks")

4. Go into the `Video` tab and make sure the video codec is set as `H.264 (X264)`. Change the `Constant Quality` to between 20–25. Smaller values will produce larger file sizes but with a higher video quality.

> 5. If you're willing to spend more time encoding slower Encoder Presets deliver better video quality and may also reduce the video file size, but do not go down to placebo as it is prohibitively slow and often performs worse than VerySlow.

Reviewer: ```suggestion
5. If you are willing to spend more time encoding, change the `Encoder Preset` under `Encoder Options`. Slower presets deliver better video quality and may also reduce video file size, but do not go down to placebo as it takes much longer than `VerySlow` for very little improvement in quality.
```

![Setting the video codec and quality in Handbrake](img/codecquality-handbrake.jpg "Setting the video codec and constant quality")

6. Set the framerate to be the same as the source and set it to constant framerate.

![Setting the video framerate in Handbrake](img/framerate-handbrake.jpg "Setting the framerate")

> 7. To resize the image of the video file, go to the `Dimensions` tab and change the width to `1280` and change the height to `720`.

Reviewer 2: Does this only downscale?

Reviewer: apparently not (anymore?). seems like these images are very outdated

![image](https://user-images.githubusercontent.com/36758269/135710860-51e49445-9281-471f-b886-f975d80a5f13.png)

Author: lol I still have the old UI for this... *same as the old image and not like above*

ig I would need to redo some of the screenshots and update the handbrake that I have locally.

Author: Looks like this new UI was released as part of the handbrake 1.4 update (dates back to July 2021).

ig it might be a good idea to pop in a notice to check for updates if the ui looks different?

3. Change the export options to help compress your file. Use `Preset` and select the quality as `Medium, 145-185 kbps`. If you want, you can enter in the metadata in the next dialog. When ready, click `OK`.

![Export settings](img/exportsettings-audacity.jpg "Export settings")

### Using FFmpeg

Sample command: ```ffmpeg -i input -c:a libmp3lame -q:a 4 -vn -sn -map_metadata -1 -map_chapters -1 output.mp3```

Explanation:

* `-i input`: your source file.

> * `-c:a libmp3lame`: specifies that you want to use the lame MP3 encoder *tbh this isn't necessary as lame should be the default*.

Reviewer: ```suggestion
* `-c:a libmp3lame`: Specify that the audio should be encoded using the LAME MP3 encoder.
```

> * `-q:a 4`: uses the same variable bitrate range as the Audacity example (lower numbers are higher bitrate). (use `-b:a 128k` for a constant 128k bitrate instead)

Reviewer: ```suggestion
* `-q:a 4`: Use the same variable bitrate range as in the Audacity example, where a lower number means higher bitrate. If you want constant bitrate, you would instead use for instance `-b:a 128k` for a constant 128kps bitrate.
```

> * `-vn -sn`: removes video & subtitles if present.
> * `-map_metadata -1 -map_chapters -1`: removes metadata & chapters if present.

Reviewer: ```suggestion
* `-vn -sn`: Remove video and subtitles if present.
* `-map_metadata -1 -map_chapters -1`: Remove metadata and chapters if present.
```
